install_config/install/prerequisites.adoc

```diff
@@ -460,11 +460,18 @@ properly when the network interface for {product-title} has `NM_CONTROLLED=no`.
 ==== Required Ports
 
 The {product-title} installation automatically creates a set of internal
-firewall rules on each host using `iptables`. However, if your network
-configuration uses an external firewall, such as a hardware-based firewall, you
-must ensure infrastructure components can communicate with each other through
-specific ports that act as communication endpoints for certain processes or
-services.
+firewall rules on each host using
+xref:../../admin_guide/iptables.adoc#overview[iptables]. However, if your
+network configuration uses an external firewall, such as a hardware-based
+firewall, you must ensure infrastructure components can communicate with each
+other through specific ports that act as communication endpoints for certain
+processes or services.
+
+[NOTE]
+====
+While iptables is the default firewall, firewalld is recommended for new
+installations. You can enable firewalld by setting `os_firewall_use_firewalld=true` in xref:../../install_config/install/advanced_install.adoc#advanced-install-configuring-firewalls[the Ansible inventory file].
+====
 
 [NOTE]
```

Comment:
@kalexand-rh - something went wrong here, I am assuming, while manually cherry picking. The note seems to be repeated.

Reply:
The other branches are correct, and I'll open a new PR to fix the `enterprise-3.9` branch.